ADMIN.adoc

```diff
@@ -0,0 +1,37 @@
+= Repository administrator responsibilities
+
+== Administrator rotation schedule
+
+Every 2 weeks a different project member assumes the role of repository administrator.
+Signing up to be an administrator is voluntary.
+To volunteer to be an administrator, contact link:https://github.com/cbudz[@cbudz] or link:https://github.com/cbudz[@bergerhoffer].
```

Comment:
The second link needs to be updated to be bergerhoffer

Reply:
@inoxx03 Would you be able to push a new commit with Andrea's requested change (earlier)?

Thanks, --Greg

Reply:
@gregraka Thank you for spotting this! My bad. Fixed in 6cd6d68
CC @bergerhoffer